.github/workflows/color-check.yml

```diff
@@ -19,7 +19,6 @@ jobs:
       - name: 📥 Checkout
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
         with:
-          fetch-depth: 0
```

Comment:
@okineadev This broke the color check pipeline. I've created a fix for it now: #2688.